main.go

```diff
@@ -76,13 +78,18 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: metricsAddr,
+		},
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "9094838c.percona.com",
-		Namespace:              ns,
```

Comment:
The `Port` and `Namespace` fields where deleted in the new version, will the manager work correctly without them?

Reply:
Tests are passing. Namespace moved to Cache.Namespaces and then renamed to Cache.DefaultNamespaces. Port is required for webhook settings and since we do not use them we do not need this. As for metrics it looks clear.